evalme/classification.py

```diff
@@ -119,7 +124,7 @@ def exact_matching_pairwise(item_gt, item_pred, label_weights=None, per_label=Fa
                                                                   per_label=per_label)
 
 
-def naive(x, y, per_label=False, label_order_matters=True, **kwargs):
+def naive(x, y, per_label=False, label_order_matters=False, **kwargs):
```

Comment:
wait, this arg already existed and was set to True? why change the default?

Reply:
because we dont want the order to matter, brah

Reply:
right, I just got the sign flipped in my head...